action.yml

```diff
@@ -28,6 +28,9 @@ inputs:
   app_build_command:
     description: "Custom command for Oryx to run when building application source code"
     required: false
+  is_static_export:
+    description: "Indicates site is static export"
```

Comment:
Can we make this consistent with the other boolean configurations, something like "Indicates site is static export **_if set to true._**"?

Also, is this only for NextJS (if so, can we mention it in the description)? And is "static export" a standard term? In searches I'm finding "Static HTML Export".